types/datetime.go

```diff
@@ -0,0 +1,281 @@
+package types
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"time"
+	"unicode"
+)
+
+type Datetime struct {
+	// Value is a timestamp in milliseconds
+	Value int64
+}
+
+func UnsafeDatetime[T int | int64 | float64](v T) Datetime {
```

Comment:
I still think this function isn't useful.

Reply:
ha! I think I stopped using it and didn't delete it. I'll get rid of it.

Reply:
Apparently I didn't stop using it. 🤔 I _will_ stop using it.